fetcher.sh

```diff
@@ -12,6 +12,7 @@ Also note that !git, !dotfiles, and !desc are different commands.
 
 !setfetch
 Distro: $NAME $ver
+Host: $(cat /sys/devices/virtual/dmi/id/product_name)
```

Comment:
Please do not directly `cat` the file here, please put it in the linux section such as

```
read -r host <  /sys/devices/virtual/dmi/id/product_name
```
then use `Host: $host` here.